project/config/pacni/config/node.type.publication.yml

```diff
@@ -7,8 +7,9 @@ dependencies:
     - origins_toc
 third_party_settings:
   menu_ui:
-    available_menus: {  }
-    parent: ''
+    available_menus:
+      - main
+    parent: 'main:'
```

Comment:
I don't think publications should be allowed in the menus.

Reply:
Committed the wrong file. This file has been removed now and the correct files added